### ISSUE TITLE
Settings of the steps in a flow cannot be perceived well

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3552]
### Describe the Bug with repro steps 

1. Create a flow with designer standalone (nx serve). 
2. Add a manual trigger. 
3. Go to settings for the flow 
4. Using keyboard, navigate to some of the (i) information icons next to settings labels. 
5. Open screen reader. Observe that tooltips are not read by screen reader. This makes it so keyboard users are not able to get information regarding settings. 

### What type of Logic App Is this happening in? 

Consumption (Portal)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/10837129/dd2e93ac-7cdf-499e-8814-7dc6962698fb>)

### Browser 

Edge

### Additional context 

Related internal work item: [https://msazure.visualstudio.com/One/\_workitems/edit/13913187](<https://msazure.visualstudio.com/One/_workitems/edit/13913187>)

AB#25622302

**Comment by Ram-G** [comment]
I verified this issue is fixed when using chome extension Screen Reader. But not when using NVDA screen reader.

Please see screenshot reference: 
![image](https://github.com/Azure/LogicAppsUX/assets/10837129/1600a854-7115-4695-8f86-ba37fde7f0e3)

@hartra344 I know that NVDA is not something you test with. So I'm looking for a fix from my end. Can we re-open this issue while the issue is not fixed with NVDA?

**Comment by hartra344** [comment]
@ram-g reopened and assigned to you.

**Comment by hartra344** [comment]
@Ram-G just going through adn trying to clean up some issues, were you ever able to verify anything on this issue?
